oteapi_dlite/strategies/function.py

```diff
@@ -67,6 +66,13 @@ class DLiteStorageConfig(AttrDict):
         None,
         description="Configuration options for the local data cache.",
     )
+    global_configuration_additions: DLiteGlobalConfiguration = Field(
```

Comment:
I think this should be considered as a quick-and-dirty solution as the strategy configurations should document the data resources or data sinks. A proper solution for pipeline-wise configurations should be sorted out in in issues https://github.com/EMMC-ASBL/oteapi-core/issues/211, 
https://github.com/EMMC-ASBL/oteapi-core/issues/177 and https://github.com/EMMC-ASBL/oteapi-core/issues/176.

Reply:
Or perhaps in an `oteapi-services` issue, really.